tox.ini

```diff
@@ -40,6 +39,15 @@ basepython = python3
 extras = dev
 commands =
     mypy -p libp2p -p examples --config-file {toxinidir}/mypy.ini
-    black --check libp2p tests examples setup.py
-    isort --recursive --check-only libp2p tests examples setup.py
-    flake8 libp2p tests examples setup.py
+    black --check libp2p tests tests_interop examples setup.py
+    isort --recursive --check-only libp2p tests tests_interop examples setup.py
+    flake8 libp2p tests tests_interop examples setup.py
+
+[testenv:py37-interop]
+deps = p2pclient
+passenv = CI TRAVIS TRAVIS_* GOPATH
+extras = test
+commands =
+    py.test tests_interop/
```

Comment:
*nitpick* The recommendation from pytest is to run the command without a dot: https://github.com/pytest-dev/pytest/issues/1629